libpf/pfelf/file.go

```
@@ -489,6 +494,27 @@ func (f *File) GetBuildID() (string, error) {
 	return getBuildIDFromNotes(data)
 }
 
+// GoVersion return the Go version if present.
+func (f *File) GoVersion() string {
```

Why not passing errors to the caller?

---

I didn't think anyone would care as the error path and non-Go path are likely to be the same but I guess swallowing errors is kinda ugly.  Happy to return (string,error) and let callers decide whether to swallow or not.